pom.xml

```diff
@@ -5,20 +5,21 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.7.5</version>
+		<version>3.0.5</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 
 	<groupId>it.gov.pagopa.rtd.ms</groupId>
 	<artifactId>rtd-ms-file-reporter</artifactId>
-	<version>1.1.0</version>
+	<version>1.1.1</version>
 	<name>rtd-ms-file-reporter</name>
 	<description>Micro-service to retrieve and keep updated the file reports for senders</description>
 
 	<properties>
-		<java.version>11</java.version>
-		<spring-cloud.version>2021.0.5</spring-cloud.version>
-		<jackson-core.version>2.14.0</jackson-core.version>
+		<java.version>17</java.version>
+		<spring-cloud.version>2022.0.2</spring-cloud.version>
+		<jackson-core.version>2.14.2</jackson-core.version>
```

Comment:
no need to use explicit version of jackson since it is already contained in spring boot parent

Reply:
I think it was specified because spring boot 2.7.x had a jackson version with CVE detected. Right now the 2.14.2 is up of date with spring boot 3.0.5